helm-chart/values.yaml

```diff
@@ -41,9 +43,21 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
+pvc:
+  storageClassName: standard
+  conf:
+    size: 1Gi
+  data:
+    size: 1Gi
```

Comment:
This will be a little bit small for the data directory as it will contain backups, cache, exports...

Reply:
It's just a default value, but I can increase it if you want :)
Don't see the ITOP recommandation

Reply:
Size increased :)